src/rastervision/cli/main.py

```diff
@@ -3,15 +3,18 @@
 import os
 
 import click
+import logging
 
 import rastervision as rv
 from rastervision.experiment import (ExperimentLoader, LoaderError)
 from rastervision.runner import (ExperimentRunner)
 from rastervision.rv_config import RVConfig
 
+log = logging.getLogger(__name__)
```

Comment:
I think that since the CLI is explicitly  for the command line, we'd want to log errors to the console with the `click.echo` method we have, and not through the logger.